### DISCUSSION TITLE
[BUG] Can't send more than [xxxx] in a single transaction

### DISCUSSION THREAD

**Discussion by sc0-0001** [Chia-Network/chia-blockchain#9169] [Support]
**Describe the bug**
Attempting to send a transaction for an amount+fee less than spendable balance, is resulting in an error.

`chia wallet show -f [fingerprint]`
> `-Spendable: 5.00530xxxxxxx xch (500530xxxxxxx mojo)`

`chia wallet send -f [fingerprint] -a 5.0 -m 0.00005 -t [address]`
> `Exception from 'wallet' {'error': "Can't send more than 27xxxxxxxxxxx in a single transaction", 'success': False}`

This should be able to successfully send and have a remaining balance of 0.00525xxxxxx XCH.

**Desktop**
- OS: Ubuntu Desktop 20.04
- Chia Version: 1.2.10

**Comment by loppefaaret** [comment]
you are hitting the limit of amount of coin ID's in a single transaction
imagine having 88 1-cent coins in your wallet, but you can only hold 37 at a time. when you go to pay 0.79 for a pack of gum, you have to take out 0.37, and hand it to the cashier, grab another 0.37, and hand it to the cashier, and finally, grab the last 0.05, and hand those to the cashier as well.
either send yourself transactions that combines all those fragmented coin ID's into a bigger coin ID, or just simply split your transaction into smaller portions

**Comment by sc0-0001** [comment]
Ah, interesting. Makes sense. Thank you for taking the time to explain. I suspect anyone farming via pool will run into similar issues.

Chia team, a more clear error message would go a long way here in easing frustration and adding clarity.

I searched this issue on Google and here in the Issues, and while I now understand one of the prior closed issues on GitHub references it, it's not clear unless you already really understand what's going on under the hood in how transactions are constructed. For the less technical farmer or user, it just seems like an arbitrary transaction limit that's a frustrating bug.

**Comment by sargonas** [comment]
Thanks for the feedback, and I agree. I'll put a pin somewhere in the idea we should revisit the messaging this returns!
